docs/source/en/llm_tutorial_optimization.md

```diff
@@ -286,7 +286,7 @@ If GPU memory is not a constraint for your use case, there is often no need to l
 For more in-detail usage information, we strongly recommend taking a look at the [Transformers Quantization Docs](https://huggingface.co/docs/transformers/main_classes/quantization#general-usage).
 Next, let's look into how we can improve computational and memory efficiency by using better algorithms and an improved model architecture.
 
-# 2. Flash Attention
+## 2. Flash Attention
```

Comment:
The missing `#` was resulting in a poorly rendered TOC on the right-hand side (see [here](https://huggingface.co/docs/transformers/llm_tutorial_optimization))